main.py

```diff
@@ -140,8 +140,8 @@ def check_signed_up(self, email):           # Returns false if there is already
     def get_schedule_for_name(self, firstname, lastname):
         schedule_data = get_schedule_data()
         for schedule in schedule_data:
-            if schedule['firstname'].lower() == firstname.lower() and \
-               schedule['lastname'].lower() == lastname.lower(): # If the schedule is the requested schedule
+            if schedule['firstname'].lower().replace(" ", "") == firstname.lower() and \
```

Comment:
This seems like something that should go in a helper function, e.g. normalize_name(firstname), which does .lower() and space replacement internally.